README.md

```diff
@@ -317,6 +317,50 @@ export default withStyles(({ color, unit }) => ({
 
 `className` and `style` props must not be used on the same elements as `css()`.
 
+#### Example with React-Router's `Link`
+[React-Router][react-router]'s [`<Link/>`][react-router-link] and `<IndexLink/>` components accept `activeClassName='...'` and `activeStyle={{...}}` as props. To mimic this functionality you can use React-Router's [`withRouter()`][react-router-with-router] HOC on top of `withStyles()`:
```

Comment:
- "React-Router" should not be hyphenated.
- Should we link to documentation for IndexLink as well?
- It might be more beginner friendly to spell out HOC instead of using an initialism.